client/state/test/index.js

```diff
@@ -24,23 +24,13 @@ describe( 'state', () => {
 	} );
 
 	describe( 'createReduxStore', () => {
-		let createReduxStore;
-
-		before( () => {
-			domEnvSetup();
-			createReduxStore = require( '../' ).createReduxStore;
```

Comment:
:+1: Nice that this turned into a simple import, I had trouble with some of the redux middleware previously